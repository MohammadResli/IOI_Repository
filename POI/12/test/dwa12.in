200
14 62 150 169 83 146 109 33 88 130 67 107 187 138 114
24 190 197 199 133 66 125 124 12 10 74 175 41 5 39 168 119 54 109 198 30 187 55 59 138
19 33 79 64 14 52 58 88 177 189 127 114 17 66 67 27 131 195 10 173
13 65 143 148 76 81 196 136 150 46 145 62 50 91
18 85 156 2 195 54 29 94 39 200 35 114 50 56 102 58 28 105 32
12 166 80 73 194 107 138 169 14 72 171 45 156
16 102 107 12 118 125 157 113 90 169 139 19 109 160 89 138 27
21 98 162 196 72 177 175 37 80 148 93 70 74 152 160 82 10 63 90 15 155 129
14 39 73 15 200 62 117 171 44 81 21 36 178 20 140
26 8 20 102 179 18 3 169 60 64 171 166 2 114 83 138 33 85 57 43 113 121 65 104 136 88 58
16 124 146 100 166 53 200 79 69 188 119 44 99 164 134 55 15
24 185 175 34 7 16 119 139 199 45 65 194 110 196 30 188 84 132 112 2 105 17 89 67 96
19 60 22 28 129 34 186 127 123 20 43 159 112 16 31 25 194 97 40 21
23 63 6 45 3 138 162 190 145 17 88 110 141 188 29 94 168 179 117 15 89 132 135 55
27 104 74 132 127 88 122 196 41 9 91 199 176 168 30 8 80 111 166 11 105 14 92 114 49 108 35 103
20 183 94 20 56 68 115 38 13 167 12 148 120 23 57 178 109 172 177 47 197
23 49 122 20 88 189 3 118 87 130 188 169 85 128 97 165 140 175 105 12 193 14 98 113
16 30 193 147 89 43 10 128 171 87 127 81 84 70 66 178 104
23 30 38 65 7 42 146 149 169 70 49 170 105 120 77 87 176 125 34 128 99 40 62 94
19 131 119 79 17 174 9 171 39 13 99 167 10 141 143 48 153 81 16 72
22 152 190 22 108 166 43 34 160 31 60 32 165 51 111 41 13 101 113 80 76 9 120
22 153 132 196 13 156 136 102 200 24 41 92 85 25 181 121 21 166 164 106 45 145 79
25 16 153 50 55 87 108 60 106 102 178 24 66 73 173 40 93 37 36 98 80 30 170 155 174 144
16 94 118 132 62 152 23 151 39 180 22 61 126 97 160 182 191
15 28 189 160 182 68 149 72 93 22 102 151 52 13 43 54
19 57 35 111 138 54 133 148 139 149 33 67 28 187 96 112 92 108 80 192
18 140 146 7 184 64 86 145 98 3 172 61 199 119 121 193 34 157 52
21 144 189 186 26 151 119 170 125 48 70 191 120 84 5 94 102 60 106 172 13 25
16 60 169 5 53 35 131 50 56 164 123 188 31 80 198 14 89
20 71 69 15 46 130 132 191 73 42 92 19 192 18 53 60 23 2 33 12 156
17 13 190 147 29 43 101 93 21 159 117 131 139 37 136 60 89 183
11 144 176 181 21 130 95 56 5 159 41 113
18 71 90 104 105 91 101 85 1 191 60 10 82 183 26 126 98 3 30
18 19 41 188 21 172 130 150 13 61 75 119 162 12 27 66 120 169 38
23 168 47 106 154 76 120 29 74 54 26 146 5 194 177 170 77 121 144 158 61 15 119 122
20 122 60 98 73 92 123 162 196 44 9 49 132 89 193 182 174 140 41 23 61
19 31 86 97 79 44 103 89 43 144 173 199 116 8 190 174 113 114 99 23
26 19 49 53 44 68 90 39 47 183 149 195 41 34 141 42 153 74 77 101 106 88 160 156 179 16 99
20 179 160 5 47 61 163 2 54 24 9 53 153 38 92 174 115 130 20 100 102
23 87 13 189 23 42 114 78 117 74 110 53 66 75 154 192 85 45 123 47 129 151 19 180
25 21 158 175 113 34 108 193 15 187 36 124 194 38 22 170 120 91 132 129 180 168 2 117 126 32
23 93 70 46 40 104 54 91 146 147 30 192 199 110 144 105 19 126 171 167 200 159 179 38
23 37 107 45 97 130 59 58 74 13 190 177 106 60 150 21 25 194 111 158 31 18 71 10
24 100 46 165 125 80 72 169 138 11 37 197 162 38 181 173 9 140 88 190 114 177 36 194 164
20 187 104 154 22 130 51 185 14 46 43 174 168 6 40 12 84 109 145 76 200
21 92 97 195 154 45 30 142 143 149 131 177 171 144 53 44 123 4 101 60 42 163
24 60 151 186 172 16 49 78 75 39 113 38 40 98 197 173 199 133 184 152 35 168 96 107 59
12 199 189 74 192 118 20 66 130 165 193 101 28
18 59 65 97 187 84 19 183 17 38 53 105 89 36 109 47 127 15 197
21 53 29 107 122 52 188 4 114 104 194 110 117 61 138 23 5 111 123 95 55 149
20 187 138 66 168 58 67 189 45 117 172 191 71 146 70 21 80 130 152 104 90
18 163 27 50 76 131 103 132 134 165 156 93 3 172 195 59 25 112 177
25 156 97 11 129 49 50 169 138 40 179 113 182 46 39 30 77 139 187 171 150 140 177 38 29 148
23 199 26 42 111 25 191 39 183 81 192 177 62 92 76 161 2 119 5 35 173 167 118 131
22 101 50 87 181 188 104 23 113 73 84 182 88 192 186 126 2 14 89 107 11 185 140
16 16 98 119 29 120 166 5 97 197 117 67 124 163 135 32 130
20 10 60 157 117 177 176 158 16 82 69 26 200 135 112 163 124 58 147 134 71
18 160 148 179 57 68 3 10 51 69 141 166 178 95 140 102 5 156 43
27 200 65 164 91 199 162 145 72 141 82 187 104 188 84 70 101 75 49 2 169 52 43 197 93 180 139 47
27 154 117 146 33 192 46 31 129 23 30 92 13 116 77 198 36 47 57 187 43 190 10 21 29 124 28 164
22 126 63 159 85 188 127 122 180 35 193 39 50 27 36 24 125 88 116 100 156 96 34
25 108 54 137 184 146 1 19 142 112 78 136 143 24 101 126 186 4 9 144 161 171 140 166 168 64
12 8 80 14 188 72 134 186 61 73 70 88 114
19 191 10 106 27 62 85 81 78 130 3 152 93 133 66 195 153 114 119 99
19 138 142 10 78 12 150 172 115 177 195 132 126 49 170 156 4 19 59 154
22 2 192 40 23 76 169 81 85 127 121 3 124 51 155 64 48 110 158 18 100 34 139
19 51 110 12 195 169 166 96 1 117 80 152 175 3 56 70 179 26 75 141
22 84 130 77 90 138 79 157 181 139 166 180 170 167 58 38 98 85 113 71 177 25 16
17 58 178 57 189 11 85 134 183 83 30 117 124 167 200 123 139 109
25 42 74 186 18 67 139 191 160 79 28 180 193 51 118 59 72 126 63 8 19 197 88 150 81 188
18 127 128 191 51 30 33 57 154 173 132 68 112 153 43 188 137 125 196
24 133 6 70 93 191 110 118 95 169 114 25 80 155 150 159 44 59 8 166 75 63 20 85 141
21 112 185 83 91 189 108 92 141 63 36 96 158 103 55 190 30 9 23 105 149 6
19 88 89 15 126 91 189 105 94 35 38 48 2 40 8 116 70 43 195 175
18 190 153 34 175 47 104 110 67 162 117 159 131 121 126 72 99 40 59
19 134 35 100 146 189 45 52 21 137 78 4 66 54 119 117 138 152 135 123
25 124 194 187 53 131 155 38 68 170 81 182 115 165 129 117 35 102 163 101 60 84 191 108 19 141
19 171 163 175 152 47 64 62 142 65 135 167 160 76 40 108 94 93 153 133
17 174 157 68 22 90 37 152 131 126 82 11 70 168 164 190 20 3
18 8 21 63 195 15 172 29 72 26 23 185 179 193 117 6 51 67 44
25 66 185 199 91 182 83 90 9 100 198 95 20 140 133 77 186 4 132 64 70 18 85 54 196 162
17 57 174 169 164 84 33 8 117 126 189 110 79 178 156 59 136 83
20 121 159 104 117 130 141 10 188 123 118 153 94 87 154 140 69 1 73 81 82
21 133 163 28 95 45 55 12 77 107 115 114 68 169 49 18 99 140 59 91 178 82
21 64 177 10 69 17 151 102 66 61 33 22 72 112 40 68 145 199 5 81 98 190
16 87 161 132 27 138 155 198 107 100 186 88 171 135 136 163 37
18 83 193 106 86 148 135 55 19 139 17 191 114 143 117 18 160 40 23
27 98 194 94 151 158 63 86 44 70 74 182 15 38 10 14 113 196 169 190 92 168 200 1 61 55 3 17
26 49 7 55 104 194 37 12 134 14 74 140 176 29 36 31 154 142 90 170 189 169 94 160 164 18 125
21 193 8 51 195 167 179 187 126 89 192 7 129 38 154 79 157 107 166 33 68 81
21 153 94 122 145 33 195 151 84 15 81 73 115 41 199 74 4 42 59 135 114 163
18 60 152 46 135 159 88 15 30 133 36 22 73 26 180 39 54 194 115
18 25 197 52 42 188 144 110 125 170 106 31 23 59 140 64 8 72 78
21 78 14 88 19 91 5 141 108 28 83 195 199 139 24 89 16 200 74 97 135 166
21 136 116 120 199 50 134 32 196 142 58 72 191 155 84 124 182 164 135 146 81 188
16 47 61 187 120 125 174 26 127 106 160 67 12 171 188 113 73
22 46 109 157 168 17 184 49 124 24 155 191 37 119 120 194 105 56 156 53 94 13 43
23 23 123 56 114 33 199 130 8 182 133 47 88 68 36 196 128 17 195 168 27 163 193 85
15 11 37 84 19 75 142 20 159 38 64 138 136 178 149 195
17 86 44 180 66 127 166 61 111 184 137 11 139 76 81 39 149 122
17 110 59 62 177 130 38 105 48 46 21 55 166 77 31 33 118 199
20 133 85 77 5 160 143 170 22 140 10 151 120 7 58 119 25 23 190 28 39
12 120 37 149 139 52 135 175 156 119 73 132 15
21 122 75 89 194 106 33 55 185 45 51 160 42 50 186 15 83 162 18 10 59 179
19 74 33 160 106 12 73 146 17 189 5 101 42 169 187 49 97 19 15 134
21 187 111 23 64 22 109 147 168 28 38 161 120 87 93 104 105 123 43 122 35 96
23 47 7 43 188 84 153 118 1 146 90 117 135 6 196 164 86 119 181 50 177 154 55 149
15 62 15 166 196 78 77 109 26 73 94 126 23 21 123 41
17 161 124 186 162 7 183 142 97 106 185 69 108 2 45 1 49 16
16 14 67 93 82 112 66 151 72 50 42 40 111 165 75 12 101
21 110 43 113 26 161 167 106 133 54 127 118 50 100 135 115 147 21 157 169 15 141
19 85 192 12 26 190 73 13 142 62 57 52 176 71 153 169 110 125 117 114
19 125 55 96 10 53 111 21 124 166 32 41 68 159 184 7 88 47 17 37
30 37 112 10 181 121 192 91 126 63 87 98 44 122 188 119 15 141 84 129 50 1 145 64 3 179 5 198 40 72 200
17 77 111 143 92 84 194 39 65 144 146 16 147 135 91 189 154 161
15 176 175 166 135 124 138 95 181 37 74 163 61 60 198 185
28 9 83 130 142 107 87 157 57 60 31 69 67 190 14 82 50 77 41 76 144 51 75 164 40 80 179 56 112
24 24 17 129 135 111 176 194 171 48 175 70 7 185 154 83 107 126 54 133 122 120 101 187 72
29 200 76 34 102 35 147 143 151 27 54 2 176 158 103 168 195 150 188 56 107 12 165 64 97 20 114 199 28 11
23 170 28 148 154 102 118 97 16 34 161 187 41 103 106 35 56 126 95 157 21 169 19 96
16 153 35 198 75 187 10 27 196 147 83 66 114 141 22 135 142
25 165 17 118 91 15 139 36 187 154 198 161 190 61 100 50 126 178 104 35 132 180 155 125 106 114
21 50 76 169 29 129 154 13 160 108 83 140 46 157 40 69 176 143 106 98 36 162
21 11 154 167 116 77 95 60 184 69 66 113 41 56 109 169 163 159 181 2 57 97
22 2 168 113 133 96 61 171 28 122 7 93 135 89 112 19 164 169 189 141 71 44 144
25 120 139 174 79 198 70 197 33 65 24 82 74 122 90 61 118 177 41 42 134 108 114 55 75 62
21 71 145 151 96 61 134 199 152 13 18 111 3 135 139 49 168 188 15 66 133 100
14 151 17 71 170 138 146 195 142 158 18 134 179 98 19
20 40 123 188 90 132 41 133 184 114 8 118 155 164 77 195 60 197 53 13 171
24 45 32 1 68 34 30 39 146 56 195 83 117 43 17 64 160 51 169 144 171 155 101 48 98
21 183 197 77 54 185 144 139 52 166 3 75 159 162 46 31 160 143 79 29 20 194
27 152 153 71 122 52 151 146 65 129 164 189 15 155 184 14 173 41 169 30 86 103 81 36 185 24 12 22
21 81 111 125 2 64 161 26 72 168 129 78 92 118 102 176 84 98 177 47 169 127
22 11 170 190 192 127 76 128 105 89 69 147 174 193 162 169 52 57 95 175 153 63 126
29 94 158 111 91 95 121 76 107 78 87 190 92 173 116 14 118 188 178 103 127 115 175 146 125 141 56 57 163 86
14 95 156 174 82 10 150 86 4 177 99 62 31 155 22
8 62 71 100 76 171 181 195 168
23 155 68 86 195 183 53 7 50 128 6 116 2 44 26 65 10 76 164 1 51 161 14 99
19 100 151 31 7 70 59 53 12 26 122 66 69 103 94 126 127 87 68 131
20 62 58 53 83 102 123 84 170 44 193 27 55 89 157 36 169 81 93 9 17
20 121 114 135 72 59 83 14 174 94 111 149 58 20 38 125 67 183 147 73 77
15 128 65 99 78 89 95 173 62 46 121 112 109 117 170 179
19 62 145 115 87 4 158 146 131 46 119 184 102 123 191 197 186 196 20 170
21 32 37 46 131 42 117 186 147 187 125 23 181 163 130 62 93 115 28 152 176 35
14 91 4 22 143 59 14 85 179 127 200 114 45 27 157
22 168 1 132 35 115 130 200 128 107 11 19 51 60 42 143 105 135 181 95 27 62 76
20 185 134 197 31 169 144 119 156 165 18 174 141 192 57 115 106 42 121 111 153
13 8 158 58 157 53 186 87 120 159 26 199 4 16
17 50 26 46 103 99 158 25 19 100 165 181 173 107 38 141 177 73
14 43 199 119 4 161 70 136 53 34 1 197 168 72 65
21 196 132 88 102 176 119 110 139 25 24 187 127 191 85 47 40 128 91 179 28 182
22 144 64 162 8 184 92 175 78 51 127 24 192 199 178 21 76 79 67 47 161 193 132
27 157 134 83 147 64 38 171 91 121 107 192 75 189 78 161 112 20 71 132 168 172 156 159 22 39 181 23
24 118 175 65 115 166 156 60 35 83 123 40 46 107 71 120 124 89 160 177 45 90 122 180 191
18 185 129 8 66 165 97 138 23 122 86 77 132 130 175 72 184 95 136
21 184 97 82 53 52 61 5 154 168 147 103 153 193 157 22 6 38 65 136 58 30
17 156 140 27 153 97 123 57 120 145 111 164 90 68 117 7 148 79
15 135 88 143 41 35 66 73 43 128 119 195 57 166 149 148
19 195 61 174 72 113 163 124 131 153 42 13 92 148 181 32 75 83 31 99
23 70 8 131 38 58 130 154 105 87 21 7 102 180 24 198 25 39 78 123 89 168 96 104
19 115 111 163 153 138 133 62 152 150 179 109 122 184 106 86 188 120 54 182
17 34 36 131 109 193 104 75 14 59 169 152 44 8 134 180 123 81
19 161 190 57 78 46 195 91 86 124 84 98 39 116 77 135 144 56 52 159
19 107 79 60 11 22 44 174 29 117 59 125 132 95 178 129 89 82 157 138
18 122 149 187 198 44 155 110 21 167 17 77 170 195 119 147 48 52 191
22 56 22 68 108 131 6 10 72 154 100 113 62 158 21 11 116 94 67 101 15 90 58
16 16 42 68 90 20 69 184 169 111 54 199 165 182 179 124 78
26 2 14 160 127 133 194 150 98 51 119 153 41 88 156 137 62 125 47 15 106 45 79 97 35 146 197
36 130 111 6 89 134 186 34 17 105 10 133 82 44 59 7 140 123 112 162 132 66 19 72 173 125 124 67 29 1 179 88 167 120 147 84 53
22 68 184 93 200 134 120 35 19 65 165 142 128 77 102 192 140 143 89 41 28 23 178
18 6 53 130 42 46 125 78 153 18 20 86 96 129 10 9 62 118 137
13 28 185 65 27 52 47 51 80 153 34 187 16 173
20 142 183 37 169 23 172 132 149 47 54 135 44 184 200 198 3 190 175 193 71
17 82 39 45 136 141 20 147 79 23 199 159 37 134 96 164 126 36
20 2 17 75 134 135 198 116 74 155 191 118 12 67 78 103 8 154 41 173 152
16 112 133 151 119 116 19 89 182 15 144 32 199 118 57 193 123
22 149 126 133 46 43 53 107 197 154 136 3 101 65 52 35 68 57 44 8 54 85 16
16 135 84 170 82 18 23 164 186 58 69 122 16 152 9 192 99
22 39 104 117 197 161 90 167 67 14 188 128 142 145 53 114 169 10 80 151 58 38 42
15 154 185 68 122 182 100 40 162 59 70 61 41 24 160 92
18 149 195 144 22 137 185 114 32 44 107 55 146 68 153 116 124 159 189
15 151 77 176 53 25 95 36 161 24 180 88 81 167 55 98
12 31 173 33 49 109 138 131 16 69 54 38 141
17 155 167 161 100 152 143 132 124 129 27 47 170 156 173 97 62 113
19 155 109 191 132 80 118 73 180 45 147 104 81 193 131 181 116 12 55 172
19 148 195 47 178 109 70 86 143 62 13 81 63 144 169 193 28 55 104 196
25 121 60 144 26 105 59 77 172 90 41 53 194 120 45 198 151 2 122 165 96 106 49 1 118 51
28 179 127 197 96 12 17 34 107 93 119 83 195 95 63 161 135 50 29 129 59 61 55 70 14 11 114 196 71
20 40 153 51 125 28 74 132 73 115 17 181 82 48 25 89 76 69 192 105 3
22 79 135 43 2 44 85 14 75 88 31 37 117 134 191 173 112 21 102 60 122 163 73
23 77 95 185 190 97 154 54 64 143 71 175 24 30 165 33 200 70 192 151 28 87 51 72
20 48 66 153 112 42 55 54 189 191 147 170 26 90 114 152 30 40 60 134 178
21 80 185 87 61 36 140 176 70 90 186 27 48 156 17 18 162 98 152 173 134 41
19 118 35 41 12 168 50 104 44 131 115 92 77 13 89 187 43 97 6 88
28 163 158 130 119 98 90 159 80 52 94 46 186 38 128 65 165 181 67 74 3 138 5 129 188 99 64 91 137
19 143 107 4 15 188 121 98 12 151 22 36 88 186 8 81 108 200 95 71
19 150 188 49 168 44 147 143 56 129 179 177 93 16 59 70 47 131 126 2
15 29 173 187 86 175 116 122 165 60 160 81 2 114 126 121
26 42 127 150 200 81 47 12 85 176 152 54 15 174 27 91 167 98 59 148 95 94 101 37 48 119 2
20 114 88 57 22 170 45 42 5 196 145 9 11 59 191 199 119 146 69 94 173
