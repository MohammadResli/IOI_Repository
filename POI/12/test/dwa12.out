103
1 2 3 5 6 7 8 11 17 18 20 21 22 27 30 34 35 36 38 40 43 44 45 47 48 51 57 58 60 63 66 67 68 70 71 73 75 76 82 84 86 88 90 91 92 93 97 98 99 101 104 105 109 111 114 115 116 124 125 127 128 130 131 132 134 136 138 141 144 145 146 147 148 149 150 151 152 154 155 156 159 160 161 164 165 166 167 172 173 176 177 179 181 185 192 193 194 195 196 197 198 199 200 
